Assets/MixedRealityToolkit/_Core/Utilities/Editor/Setup/EnforceEditorSettings.cs

```diff
@@ -148,9 +166,38 @@ private static bool IsNewSession()
             return false;
         }
 
+        private static bool FindDirectory(string directoryPathToSearch, string directoryName, out string path)
```

Comment:
I suppose I could make this an extension, but this class is supposed to be self sufficient as it runs before all other assemblies are compiled by Unity.

Reply:
I think this is fine here.